packages/browser-client/src/peerActions.ts

```diff
@@ -81,15 +81,15 @@ export class PeerActions {
       const block = reassembleBlock(header!.value as Uint8Array, undefined)
       return block //
     } else if (type === 'body') {
-      const headerKey = HistoryNetworkContentKeyUnionType.serialize({
+      const headerKey = HistoryNetworkContentKeyType.serialize({
         selector: 0,
         value: {
           chainId: 1,
```

Comment:
Do we need to remove chainID here as well?  I'm assuming we never use chainId anywhere, right?

Reply:
Ah, this is outdated.  content keys are not ssz anymore.

Reply:
Fixed -- Same thing now looks like
```
const headerContentId = getHistoryNetworkContentId(BlockHeader, this.state.blockHash)